src/net/server/mod.rs

```diff
@@ -95,6 +109,68 @@ impl IntoResponse for MpcHelperServerError {
     }
 }
 
+/// Provides a mapping of [`QueryId`]s to senders that forward data to a [`Network`]. Every time a
+/// new query is started, a [`Network`] is created to handle communication for that query. When the
+/// server receives a request, it must know which [`Network`] to forward that request to, so it
+/// holds this mapping to accomplish that.
+///
+/// Is shareable by `clone()`ing.
+#[derive(Clone)]
+pub struct MessageSendMap {
```

Comment:
I thought it is not stacked on 225. Is it a different `MessageSendMap` implementation?

Reply:
honestly, this might have been a mistake on my part. i didn't intend for this to be stacked. Let me open a PR with just the stacked part of the diff